src/plugins/intel_gpu/src/plugin/transformations_pipeline.cpp

```diff
@@ -738,7 +738,7 @@ void TransformationsPipeline::apply(std::shared_ptr<ov::Model> func) {
         manager.register_pass<ov::intel_gpu::KVCacheFusion>();
         manager.register_pass<ov::intel_gpu::FullyConnectedConvertFusion>();
         if (!device_info.supports_immad) {
-            manager.register_pass<ov::intel_gpu::TransposeMatMulFusion>();
+            // manager.register_pass<ov::intel_gpu::TransposeMatMulFusion>();
```

Comment:
Is it intended?

Reply:
Oh!, no it's my fault. Thank you!

Reply:
I remove the test code and added unit tests for the newly added osv32 kernel.